### PR TITLE
CDAP-3148 Create temp dir in master's local data dir

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-3148

### CODE REVIEW

**Comment by chtyim** [comment]
LGTM

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java [hunk 1]
```diff
@@ -439,13 +445,13 @@ private static File hijackHiveConfFile(File confFile) {
 
     // we prefer jars at container's root directory before job.jar,
     // we edit the YARN_APPLICATION_CLASSPATH in yarn-site.xml using
-    // co.cask.cdap.explore.service.ExploreServiceUtils.hijackYarnConfFile and
+    // co.cask.cdap.explore.service.ExploreServiceUtils.updatekYarnConfFile and
```
Comment (gsps1) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java]: typo

**Comment by poornachandra** [comment]
Build - http://builds.cask.co/browse/CDAP-RBT393-1

**Comment by poornachandra** [comment]
@gsps1 Fixed typo. Please check.

**Comment by gsps1** [comment]
LGTM

**Comment by poornachandra** [comment]
Build - http://builds.cask.co/browse/CDAP-RBT393-2

**Comment by sreevatsanraman** [comment]
Build is :green_apple: merging.

**Pull request merged**